### DISCUSSION TITLE
Poor SSB RX

### DISCUSSION THREAD

**Discussion by Zzegfried** [egzumer/uv-k5-firmware-custom#369] [General]
https://www.youtube.com/watch?v=EzuDTiOunnY

This ham guy has compared the SSB RX of the egzumer firmware with the IJV and Fagci firmwares. The IJV and Fagci are similar in performance, but both are much better than the egzumer firmware even though it is based on Fagci.

**Comment by pietss** [comment]
> https://www.youtube.com/watch?v=EzuDTiOunnY
> 
> This ham guy has compared the SSB RX of the egzumer firmware with the IJV and Fagci firmwares. The IJV and Fagci are similar in performance, but both are much better than the egzumer firmware even though it is based on Fagci.

It was talked about here https://github.com/egzumer/uv-k5-firmware-custom/discussions/358

I think I will personally do a comparison in a fortnight with two UV-K5 radios, maybe I didn't check carefully and therefore I'm wrong but I think there aren't many comparison tests between the various firmwares.

**Reply by Zzegfried** [reply]
Ah, sorry, I missed that. I'll close this thread.

**Comment by AsciiWolf** [comment]
@egzumer Any chance this discussion thread could be re-opened (or #358 renamed to point out that it is about the SSB reception)? Thanks!

**Reply by egzumer** [reply]
There is more content there, renamed.

**Reply by AsciiWolf** [reply]
Thanks!
